pkg/scorecard_raw_results.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkg
+
+import (
+	"time"
+
+	"github.com/ossf/scorecard/v3/checker"
+)
+
+// ScorecardRawResult struct is returned on a successful Scorecard run.
+type ScorecardRawResult struct {
```

Comment:
This struct is the same as `ScorecardResult` struct. Can we use the old struct itself? Or maybe simply add a `*RawResult` ptr to the old struct?

Reply:
I tried to keep everything separate while we transition. Shall I try to merge them now or later?

Reply:
IMO we can merge them now since the diff is minimal.

Reply:
done.